### PR TITLE
Move CRC search to magic

### PR DESCRIPTION
**Pull request by sylvainpelissier**
- [x] Mark this if you consider it ready to merge
- [x] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**
Move the CRC32 search from `/ck` to `/m` since it is not really Crypto material. In addition, the big/little endian selector does not really applies since a binary can contain both.

### CODE REVIEW

**Comment by trufae** [comment]
So i like to have it in /m but lrobably i expect it to be in /ck too wdyt?

**Comment by sylvainpelissier** [comment]
> /ck is faster than /m

I wonder why `/m` is so slow even when using a single pattern file.

Yes sure we can keep them both specially right now that `/m` is slow.

**Comment by sylvainpelissier** [comment]
I've revert the `\ck` modifications.

File: libr/core/cmd_search.inc.c [hunk 1]
```diff
@@ -4338,8 +4338,8 @@ static int cmd_search(void *data, const char *input) {
 
 				// crc32
 				kw = (le)
-					? r_search_keyword_new_hexmask ("00000000963007772c610eeeba51099919c46d078ff46a7035a563e9a395649e3288db0ea4b8dc791ee9d5e088d9d2972b4cb609bd7cb17e072db8e7911dbf90", NULL)
-					: r_search_keyword_new_hexmask ("0000000077073096ee0e612c990951ba076dc419706af48fe963a5359e6495a30edb883279dcb8a4e0d5e91e97d2d98809b64c2b7eb17cbde7b82d0790bf1d91", NULL);
+						? r_search_keyword_new_hexmask ("00000000963007772c610eeeba51099919c46d078ff46a7035a563e9a395649e3288db0ea4b8dc791ee9d5e088d9d2972b4cb609bd7cb17e072db8e7911dbf90", NULL)
```
Comment (trufae) [inline libr/core/cmd_search.inc.c:4341]:
```suggestion
					? r_search_keyword_new_hexmask ("00000000963007772c610eeeba51099919c46d078ff46a7035a563e9a395649e3288db0ea4b8dc791ee9d5e088d9d2972b4cb609bd7cb17e072db8e7911dbf90", NULL)
```

File: libr/core/cmd_search.inc.c [hunk 2]
```diff
@@ -4338,8 +4338,8 @@ static int cmd_search(void *data, const char *input) {
...
+						: r_search_keyword_new_hexmask ("0000000077073096ee0e612c990951ba076dc419706af48fe963a5359e6495a30edb883279dcb8a4e0d5e91e97d2d98809b64c2b7eb17cbde7b82d0790bf1d91", NULL);
```
Comment (trufae) [inline libr/core/cmd_search.inc.c:4342]:
```suggestion
					: r_search_keyword_new_hexmask ("0000000077073096ee0e612c990951ba076dc419706af48fe963a5359e6495a30edb883279dcb8a4e0d5e91e97d2d98809b64c2b7eb17cbde7b82d0790bf1d91", NULL);
```

**Pull request merged**